frontend/model/colors.js

```diff
@@ -0,0 +1,59 @@
+export default {
+  'blue': {
+    'name': 'blue',
+    'theme': 'light',
+    'primary': '#5dc8f0',
+    'primarySaturated': '#5dc8f0',
+    'primaryText': '#17b0ea',
+    'primaryTextHover': '#fff',
+    'secondary': '#a0d10e',
+    'tertiary': '#f89201',
+    'danger': '#FC6F6F'
+  },
+  'green': {
+    'name': 'green',
+    'theme': 'light',
+    'primary': '#c2d879',
+    'primarySaturated': '#90BD0D',
+    'primaryText': '#ffffff',
+    'primaryTextHover': '#363636',
+    'secondary': '#a0d10e',
+    'tertiary': '#f89201',
+    'danger': '#FC6F6F'
+  },
+  'orange': {
+    'name': 'orange',
+    'theme': 'light',
+    'primary': '#f8b85d',
+    'primarySaturated': '#F89302',
+    'primaryText': '#17b0ea',
+    'primaryTextHover': '#fff',
+    'secondary': '#a0d10e',
+    'tertiary': '#f89201',
+    'danger': '#FC6F6F'
+  },
+  'silver': {
+    'name': 'silver',
+    'theme': 'light',
+    'primary': '#adadad',
+    'primarySaturated': '#F89302',
+    'primaryText': '#17b0ea',
+    'primaryTextHover': '#fff',
+    'secondary': '#a0d10e',
+    'tertiary': '#f89201',
+    'danger': '#FC6F6F'
+  },
+  'dark': {
+    'name': 'dark',
+    'theme': 'dark',
+    'primary': '#5dc8f0',
+    'primaryBG': '#363636',
+    'primaryBGA': 'rgba(244, 243, 244, 0.1)',
+    'primarySaturated': '#5dc8f0',
+    'primaryText': '#ffffff',
+    'primaryTextHover': '#363636',
+    'secondary': '#a0d10e',
+    'tertiary': '#f89201',
+    'danger': '#FC6F6F'
+  }
+}
```

Comment:
Do these need to be defined in the SASS/SCSS (maybe a rollup plugin can help here)? I.e. so that they're defined in a single location and accessible from both JS and SASS/SCSS?

(Or am I not understanding how it works?)

Reply:
This is defined only in a single location.
It make more sense to have data in a json or js file than in css.

Reply:
Knowing that, do you still want this to be in SCSS file?

Reply:
> Knowing that, do you still want this to be in SCSS file?

@pieer no, as long as it's only in one canonical location it's totally fine and you decide what's better 😄 

I'll continue with my review of this PR, these were just some preliminary questions I had. I'm feeling better, cold seems to be going away 😄